src/main/java/com/amazon/dlic/auth/http/saml/Saml2SettingsProvider.java

```diff
@@ -133,6 +136,7 @@ private boolean isUpdateRequired() {
     private void initMisc(HashMap<String, Object> configProperties) {
         configProperties.put(SettingsBuilder.STRICT_PROPERTY_KEY, true);
         configProperties.put(SettingsBuilder.SECURITY_REJECT_UNSOLICITED_RESPONSES_WITH_INRESPONSETO, true);
+        configProperties.put(SettingsBuilder.SP_PRIVATEKEY_PROPERTY_KEY, this.spSignaturePrivateKey);
```

Comment:
Curious where we are using spSignaturePrivateKey in the Saml2SettingsProvider.

Reply:
It's read from the settings here: https://github.com/onelogin/java-saml/blob/master/core/src/main/java/com/onelogin/saml2/authn/SamlResponse.java#L1074

Reply:
Thanks for clarifying